modules/ROOT/pages/deploy-to-cloudhub.adoc

```diff
@@ -98,7 +98,10 @@ The following table shows the available parameters to configure the CloudHub dep
 If not set, by default this value is set to +https://anypoint.mulesoft.com+. | No
 | `muleVersion` | The Mule runtime engine version to run in your CloudHub instance.  +
 Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
-Example values: `4.3.0`, `4.2.2-hf4` | Yes
+Example value: `4.3.0` +
+
+Starting with Mule 4.5, deployments to CloudHub require Major.Minor version which deploys the latest version of Mule runtime. 
```

Comment:
Sorry! I missed one comma. It should read:
"Starting with Mule 4.5, deployments to CloudHub require Major.Minor version, which deploys the latest version of Mule runtime".